CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## v1.5.0
+(Just for temporary reference, this is the last commit considered: 71566ae832ae120037a31591ba0b72305a6f3b2a)
+
+In this new release of AiiDA the support for python 3.5 is dropped [[#4386]](https://github.com/aiidateam/aiida-core/pull/4386).
+This version remains compatible with all current python versions that are not end-of-life:
+ * 3.6
+ * 3.7
+ * 3.8
+
+### Features
+- REST API: list endpoints at base URL [[#4412]](https://github.com/aiidateam/aiida-core/pull/4412)
+- `ProcessBuilder`: allow unsetting of inputs through attribute deletion [[#4419]](https://github.com/aiidateam/aiida-core/pull/4419)
+- `verdi migrate`: make `--in-place` work across different file systems [[#4393]](https://github.com/aiidateam/aiida-core/pull/4393)
+
+### Improvements
+- Refactor Import Archive [[#4510]](https://github.com/aiidateam/aiida-core/pull/4510)
```

Comment:
```suggestion
- Refactor the archive reading code from the import implementation [[#4510]](https://github.com/aiidateam/aiida-core/pull/4510)
```

Reply:
I changed this based on this recommendation, but I thought it would be better to adapt it to the format of the rest of the archive restructure logs. See first line:

```
- Archive importer: refactor the reading code [[#4510]](https://github.com/aiidateam/aiida-core/pull/4510)
- Archive importer: refactor the migrations code [[#4532]](https://github.com/aiidateam/aiida-core/pull/4532)
- Archive exporter: refactor progress bar [[#4504]](https://github.com/aiidateam/aiida-core/pull/4504)
- Archive exporter: decouple the extraction of data from the writing to file [[#4448]](https://github.com/aiidateam/
```